Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+# gem 'sqlite3'
+gem 'pg', '~> 0.20.0'
```

Comment:
Suggestion: allow us to switch between SQLite and Postgres, since not all of us have a Postgre server at hand.